Dockerfile

```diff
@@ -1,31 +1,27 @@
 FROM tutum/apache-php
+ENV DEBIAN_FRONTEND noninteractive
 
 # Install mysql-client
-RUN apt-get update && apt-get -y upgrade
-RUN apt-get install -y mysql-client unzip
-ENV TERM xterm
+RUN apt-get update && apt-get install -y --no-install-recommends \
```

Comment:
I would prefer that on multiple RUN for readability. Is there a reason you made it a single command? Also any reason you removed the `upgrade` part?